File: src/com/l2kt/gameserver/model/actor/status/PlayableStatus.java
package com.l2kt.gameserver.model.actor.status;

import com.l2kt.gameserver.model.actor.Playable;

public class PlayableStatus extends CreatureStatus
{
	public PlayableStatus(Playable activeChar)
	{
		super(activeChar);
	}
	
	@Override
	public Playable getActiveChar()
	{
		return (Playable) super.getActiveChar();
	}
}

File: src/com/l2kt/gameserver/model/actor/status/PlayableStatus.kt
package com.l2kt.gameserver.model.actor.status

import com.l2kt.gameserver.model.actor.Playable

open class PlayableStatus(activeChar: Playable) : CreatureStatus(activeChar) {

    override val activeChar: Playable
        get() = super.activeChar as Playable
}